tools/accuracy_checker/accuracy_checker/annotation_converters/super_resolution_converter.py

```diff
@@ -94,12 +114,17 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
             lr_file_name = lr_file.parts[-1]
             upsampled_file_name = self.upsample_suffix.join(lr_file_name.split(self.lr_suffix))
             if self.two_streams and self.generate_upsample:
-                self.generate_upsample_file(lr_file, self.upsample_factor, upsampled_file_name)
+                dest_path = self.upsampled_dir if self.upsampled_dir else None
+                self.generate_upsample_file(lr_file, self.upsample_factor, upsampled_file_name, dest_path)
             hr_file_name = self.hr_suffix.join(lr_file_name.split(self.lr_suffix))
             if check_content:
-                if not check_file_existence(self.data_dir / hr_file_name):
+                hr_dir = self.hr_dir if self.hr_dir else self.data_dir
+                if not hr_dir:
+                    content_errors.append('No one of the data_dir or hr_dir parameters are provided')
+                if hr_dir and not check_file_existence(hr_dir / hr_file_name):
                     content_errors.append('{}: does not exist'.format(self.data_dir / hr_file_name))
-                if self.two_streams and not check_file_existence(self.data_dir / upsampled_file_name):
+                upsampled_dir = self.upsampled_dir if self.upsampled_dir else lr_dir
+                if self.two_streams and not check_file_existence(upsampled_dir / upsampled_file_name):
                     content_errors.append('{}: does not exist'.format(self.data_dir / upsampled_file_name))
 
             identifier = [lr_file_name, upsampled_file_name] if self.two_streams else lr_file_name
```

Comment:
here need to be more accurate. If upsample and lr are stored in different directories